python/test/unit/language/test_line_info.py

```diff
@@ -212,3 +212,22 @@ def test_line_info_interpreter(func: str):
         expected_def_lineno = 68
     kernel.rewrite()
     assert kernel.rewriter.def_file_lineno == expected_def_lineno
+
+
+@pytest.mark.parametrize("disable_line_info", ["0", "1"])
+def test_line_info_env(disable_line_info: str):
+    if is_interpreter():
+        pytest.skip("interpreter does not support warmup compilation")
+
+    try:
+        obj_kind, command, anchor, separator = get_disassembler_command_and_debug_line_format()
+    except BaseException:
+        pytest.skip("disassembler is not available")
+
+    shape = (128, )
+    import os
+    os.environ["TRITON_DISABLE_LINE_INFO"] = disable_line_info
```

Comment:
This will change the env variable globally for other tests right?
Can we do something like this:
https://github.com/triton-lang/triton/blob/main/python/test/unit/test_debug_dump.py#L10

Reply:
You're right I didn't notice this test. Thanks for the tip